src/library_gl.js

```diff
@@ -3817,6 +3856,30 @@ var LibraryGL = {
 #endif
   },
 
+#if GL_STATE_CACHE
```

Comment:
I see. Perhaps add a comment explaining this?

Another question, does this not interfere with the glProcAddress stuff, for which we already have another alias for each GL call (I think it was also `emscripten_` or similar)? Or is that why this conflicts with gl emulation?